### ISSUE TITLE
[Reserved] - Bot All Contributors

### ISSUE THREAD

**Issue by mazipan** [phpid-jakarta/phpid-mentoring-frontend#1]

**Comment by mazipan** [comment]
@all-contributors please add @mazipan for code, maintenance, and design

**Comment by mazipan** [comment]
@all-contributors please add @mazipan for code, maintenance, bugs, and design

**Comment by ekojs** [comment]
ini maksudnya gimana bang @irfan-maulana-tkp @mazipan ?😂

**Comment by mazipan** [comment]
Nope, ini kerjaan buat bantuin bikin landing page dan registrasi buat program terbaru PHPID berkaitan dengan mentoring.
Saya dan om Peter akan working on this.
